File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/ShowAnswerMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.question.AnswerDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ShowAnswerMessage extends BaseMessage {
    private AnswerDto answer;

    public ShowAnswerMessage() {
        super(MessageType.SHOW_ANSWER);
    }
}

File: front/src/app/core/domain/message/show-answer.message.ts
import {GamePlayerResultDto} from "@domain/games/game-player-result.dto";
import {BaseMessage} from "@domain/message/base.message";
import {AnswerDto} from "@domain/questions/answer.dto";
import { MessageTypeEnum } from "./message-type.enum";

export class ShowAnswerMessage extends BaseMessage {
    override readonly type = MessageTypeEnum.SHOW_ANSWER;
    answer: AnswerDto;
}
